### PR TITLE
FEAT: Alert css

### PR DESCRIPTION
**Pull request by vivekmehto**
https://user-images.githubusercontent.com/76930847/152107376-f76ad1fb-3356-4c8c-a239-df9b3c0b33f0.mov

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.alert {
+  display: flex;
+  align-items: center;
+  margin: 0.5rem 1rem;
+  padding: 0.5rem 1rem;
+  border-radius: 5px;
+  width: 100%;
+  border: 1px var(--grey) solid;
```
Comment (vanshitalavange) [inline components/alert/alert.css:8]: Didn't know that we can provide value to the border property in this way as well.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.alert-danger {
+  background-color: var(--red-l);
+}
+
+.alert-warning {
+  background-color: var(--yellow-l);
+  color: var(--black);
+}
```
Comment (vanshitalavange) [inline components/alert/alert.css:56]: Good work, you included a lot of variations.

**Review by tanishq20 — COMMENTED** [review]
Good to merge.

File: components/alert/alert.css (hunk 1 above)
Comment (vivekmehto) [reply components/alert/alert.css:8]: Yes, we can.

File: components/alert/alert.css (hunk 2 above)
Comment (vivekmehto) [reply components/alert/alert.css:56]: Thank You.

**Pull request merged**